test/benchee/formatters/csv_integration_test.exs

```diff
@@ -26,7 +28,7 @@ defmodule Benchee.Formatters.CSVIntegrationTest do
         Benchee.run(
           %{
             "Sleep" => fn -> :timer.sleep(10) end,
-            "Sleep longer" => fn -> :timer.sleep(20) end
+            "List" => fn -> [:rand.uniform()] end
```

Comment:
why did we change this?

Reply:
I wanted something that would have some memory usage. The `timer.sleep` calls all have 0 memory usage.

Reply:
:ok_hand: